elasticapm/base.py

```diff
@@ -178,6 +178,12 @@ def capture(self, event_type, date=None, context=None, custom=None, stack=None,
         if data:
             url = self.config.server_url + constants.ERROR_API_PATH
             self.send(url, **data)
+            c = self.config.transaction_mark_errors
```

Comment:
Maybe it would be easier to just overlay the errors in the UI using the errors index, instead of "duplicating" the error events in marks? WDYT?

Reply:
Probably (although it would need another ES query to get that data). However, this is just an experiment, hence the `[WIP]` marker, and mostly was intended as a source of example data for the frontend team. I have no intention of merging it anytime soon :)